src/ui/MessageInput/index.jsx

```diff
@@ -420,6 +420,7 @@ const MessageInput = React.forwardRef((props, ref) => {
               e.preventDefault();
             } else {
               if (!e.shiftKey && e.key === MessageInputKeys.Enter
+                && textField?.textContent?.trim().length > 0
```

Comment:
This newly added line is the only one directly related to "prevent white spaces only sending by hitting the enter key"